tests/test_etcd.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import logging
+from nose.plugins.attrib import attr
+logger = logging.getLogger(__file__)
+
+from tests.common import load_check
+
+@attr(requires='etcd')
+class TestEtcd(unittest.TestCase):
```

Comment:
That would be amazing if you can re-use the new common class AgentCheckTest in `tests.common` (you can look at rabbitmq and riakcs). Again I can help there.